app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
@margOnline if it needs to be opened up to support the wider WPCC I'm sure we can create a card for further improvements which doesn't have to be on this PR.